lib/kuzzle/kuzzle.js

```diff
@@ -247,6 +251,60 @@ class Kuzzle extends KuzzleEventEmitter {
     process.exit(0);
   }
 
+  /**
+   * Execute multiple handlers only once on any given environment
+   * 
+   * @param {Array<{ id: string, handler: () => void, description?: string }>} installations - Array of unique methods to execute
+   * 
+   * @returns {Promise<void>}
+   */
+  async install (installations) {
+    if ( !installations || !installations.length) {
+      return;
+    }
+    
+    const mutex = new Mutex('backend:installations');
+    await mutex.lock();
+
+    try{
+      for (const installation of installations) {
+        const alreadyInstalled = await this.ask(
+          'core:storage:private:document:exist',
+          'kuzzle',
+          'installations',
+          installation.id );
+
+        if (!alreadyInstalled) {
```

Comment:
```suggestion
        if (! isAlreadyInstalled) {
```

Reply:
I know better would be `wasAlreadyInstalled` but let's say I like **is** better. :wink: